src/app/components/LinkedData/types.ts

```diff
@@ -32,7 +32,7 @@ export interface LinkedDataProps {
   aboutTags?: Tag[];
   entities?: object[];
   imageLocator?: string;
-  bylineLinkedData?: BylineLinkedData | null;
+  bylineLinkedData?: (BylineLinkedData | null)[];
```

Comment:
I don't think we should have an array of nulls
```suggestion
  bylineLinkedData?: BylineLinkedData[] | null;
```

Reply:
this is correct for my current implementation , the map in bylineExtractor would return [null] as it returns null when theres no authorname - but i agree actually thinking about this, i will fix it to return just null instead of an array of null if this happens 👍